infrastructure/variables.tf

```diff
@@ -99,3 +99,7 @@ variable "capacity" {
 variable "common_tags" {
   type = "map"
 }
+
+variable "website_local_cache_sizeinmb" {
```

Comment:
could you add a comment about why you're overriding the default module-webapp setting? in case someone else looks at this